### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by MichaelAldrich**
Cleanup from repo splits.
Update docs build.
Lots of doc strings

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Welcome to the nxt api documentation!
+=====================================
+
+The content within this documentation is the "public" api, what nxt's version numbers are constrained to.
+
+These documents represent the supported parts of the api. More pieces
+will be documented as python ergonomics improve. 
+In the source there are many more classes, methods, and even a few modules not 
+listed here. 
+Any use of those methods is at your own risk.
```
Comment (MichaelAldrich) [inline docs/source/index.rst:15]: Source of the homepage, most valuable text to review

File: nxt/tokens.py [hunk 2]
```diff
@@ -81,6 +96,8 @@ def get_standalone_tokens(raw_value, token_types=TOKENTYPE.ALL):
     else:
         if not raw_value or not isinstance(raw_value, str):
             return []
+    if token_types is None:
+        token_types = TOKENTYPE.ALL
```
Comment (MichaelAldrich) [inline nxt/tokens.py:100]: A code change that snuck in. I made this change in order to clean up the messy argument expansion sphinx was doing with `TOKENTYPE.ALL` as the default kwarg value. Not my favorite change ever.

Comment (ImLucasBrown) [reply nxt/tokens.py:100]: Hmmm its fine I guess. I know we use defaults like that in a few other places, I really don't want to have to change all those other signatures like this.

**Review by ImLucasBrown — APPROVED** [review]
The only thing I noticed is the details at the top of `runtime` should maybe be a little more clear that those methods are ones accessible by nodes in their code block. _Not_ the actual tooling used to run a graph.

**Pull request merged**